resources/kyma-environment-broker/values.yaml

```diff
@@ -150,3 +150,63 @@ e2e:
     timeout: "100m"
     preUpgradeVersion: ""
     targetVersion: "master-156dc1a5"
+
+global:
+  kyma_environment_broker:
+    secrets:
+      integrationSystemCredentials:
+        name: compass-kyma-environment-broker-credentials
+  istio:
+    gateway:
+      name: "compass-istio-gateway"
+      namespace: "compass-system"
+    proxy:
+      port: 15020
+
+  isLocalEnv: false
+
+  gateway:
+    tls:
+      host: compass-gateway
+      secure:
+        oauth:
+          host: compass-gateway-auth-oauth
+  oauth2:
+    host: oauth2
+
+  images:
+    containerRegistry:
+      path: eu.gcr.io/kyma-project/incubator
+    kyma_environment_broker:
+      dir:
+      version: "PR-1434"
+    schema_migrator:
+      dir:
+      version: "PR-1425"
+    tests:
+      e2e_provisioning:
+        dir:
+        version: "9103d980"
+
+  defaultTenant: 3e64ebae-38b5-46a0-b1ed-9ccee153a0ae
+
+  provisioning:
+    enabled: true
```

Comment:
I do not think this value still makes sense. The registration job is labeled with it but the whole chart will still be installed and broker will just not work.